code/datums/emergency_calls/cryo_marines.dm

```diff
@@ -12,11 +12,14 @@
 	name_of_spawn = /obj/effect/landmark/ert_spawns/distress_cryo
 	shuttle_id = ""
 	var/leaders = 0
+	var/deployed_marines = 0
 
-/datum/emergency_call/cryo_squad/spawn_candidates(announce, override_spawn_loc)
+/datum/emergency_call/cryo_squad/spawn_candidates(announce, override_spawn_loc, announce_dispatch_message)
```

Comment:
what's the extra arg for?

Reply:
brings us up to speed with the parent /datum/emergency_call/spawn_candidates()